docs/integrations/azure/locations.md

```diff
@@ -0,0 +1,30 @@
+---
+id: locations
+title: Azure DevOps Locations
+sidebar_label: Locations
+description: Documentation on Azure DevOps location integration
+---
+
+The Azure integration supports loading catalog entities from Azure DevOps.
+Components can be added to
```

Comment:
```suggestion
Entities can be added to
```

i know there's a discussion to be had whether "Entity" is a particularly suitable term to throw around especially for newcomers. But I think Component is almost equally generic and, in any case, just plain too specific.